From 8b68976aae482794e54e02444ee1a22676d1a00e Mon Sep 17 00:00:00 2001
From: xuxiang <xuxiang@eswincomputing.com>
Date: Thu, 13 Jun 2024 09:53:27 +0800
Subject: [PATCH 088/109] WIN2030-15279:fix: mcu check HSE clk readybit fail

Changelogs:
1. when SystemClock_Config, MCU check HSE clock ready flag fail,  resulting in failure of system initialization
2. If MCU system Clock configuration, the board is considered an exception

Change-Id: I575a56f926daebb0281da77174a5ed0468789d80
---
 Core/Src/hf_board_init.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/Core/Src/hf_board_init.c b/Core/Src/hf_board_init.c
index 9132ede..9d515ec 100644
--- a/Core/Src/hf_board_init.c
+++ b/Core/Src/hf_board_init.c
@@ -182,9 +182,7 @@ static void SystemClock_Config(void)
 	RCC_OscInitStruct.PLL.PLLN = 336;
 	RCC_OscInitStruct.PLL.PLLP = RCC_PLLP_DIV2;
 	RCC_OscInitStruct.PLL.PLLQ = 7;
-	if (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK) {
-		Error_Handler();
-	}
+	while (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK);
 
 	/** Initializes the CPU, AHB and APB buses clocks
 	 */
-- 
2.25.1

